system/assets/Gruntfile.js

```diff
@@ -80,6 +80,7 @@ module.exports = function( grunt ) {
 				},{
 					  src  : [
 					  	"js/admin/lib/plugins/jquery.dataTables.js", // must come first
+					  	"js/admin/lib/plugins/jquery.moment.js", // must come first
 					  	"js/admin/lib/plugins/*.js"
 					  ]
 					, dest : "js/admin/lib/plugins-1.0.007.min.js"
```

Comment:
@josephpbuarao Could you bump the version number of this up to plugins-1.1.000.min.js? This ensures we don't have any browser caching issues. After bumping the version + running `grunt all`, you'll need to manually delete the old `plugins-1.0.007.min.js` file.